### ISSUE TITLE
Fix VPin MAME ALTSound Settings 

### ISSUE THREAD

**Issue by syd711** [syd711/vpin-studio#319]
The value is not a boolean. See documentation:

**Sound Mode:**
0 = standard built-in PinMAME emulation

1 = built-in alternate sound file support
(store the alternate sound files in a PinSound-like directory structure (incl. text files for ducking and gain),
within a new PinMAME directory subfolder ‘altsound’ and there within the machines shortname subfolder.
Ex. for ij_l7: ‘C:\PinMAME\altsound\ij_l7’
OR for an example alternate sound file for tz_94h: ‘C:\PinMAME\altsound\tz_94h\jingle\000064-load_gumball_pt_1\load_gum__LEGACY.ogg’)

[this was the initial implementation and the option was retained basically because of inertia, I think. It works, so why take it out, etc.]

2 = external pinsound

[this is the latest implementation that bridges between PinSound Studio and VPinMAME. Through PinSound Studio you get the same features and functionality of their hardware solution including individual gain controls for each channel. I have not messed around too much with the channel gains. This does require that PinSound Studio run in the background. For PinballX users, you would either just always keep it running in the background, or you will need to code a launch script in AutoHotkey to manage the different programs. PinSound Studio was not originally coded for virtual pinball users. It always starts maximized, so you would need your script to wait until PinSound Studio is active and then minimize the window. ]

3 = external pinsound + psrec sound recording
(must have PinSound Studio 0.7 or newer running at the same time:
make sure that the Windows permissions match between PinSound Studio and Visual PinMAME/Pinball,
In other words, if one is using admin permissions when running, then both need to be started with that)

**Comment by syd711** [comment]
Version 2.21.2 has been released and contains the fix.
